WooCommerce/Classes/System/WooConstants.swift

```diff
@@ -83,12 +83,12 @@ extension WooConstants {
         ///
         case productsFeedback = "https://automattic.survey.fm/woo-app-feature-feedback-products"
 
-        /// URL for the shipping labels M1 feedback survey
+        /// URL for the shipping labels M3 feedback survey
         ///
 #if DEBUG
-        case shippingLabelsRelease1Feedback = "https://automattic.survey.fm/woo-app-testing-feature-feedback-shipping-labels"
+        case shippingLabelsRelease3Feedback = "https://automattic.survey.fm/woo-app-testing-feature-feedback-shipping-labels"
```

Comment:
Just checking, is it ok that we are linking to the same survey that we used for M1 features? Or do we need to be able to distinguish between people on older app versions sending in feedback about M1 features vs. M3 features? (I figure it's probably ok but wanted to point out that all the feedback will go to the same place.)

Reply:
It's ok, because we send some query items in the URL, like the milestone https://github.com/woocommerce/woocommerce-ios/pull/4628/files#diff-5ae6bdf31bfc7ab2a1149875d68fb54156193b998854de8c4afb472b1bcfef85R87